arpack-ng-config.cmake.in

```diff
@@ -19,7 +19,7 @@ set(includedir "@includedir@")
 
 # Create arpack targets.
 add_library(ARPACK::ARPACK INTERFACE IMPORTED)
-set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${includedir}/arpack")
-set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_LINK_LIBRARIES      "arpack")
+set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${includedir}/arpack${SUFFIX64}")
```

Comment:
This should be `@SUFFIX64`, not `${SUFFIX64}`.

Reply:
For sake of completeness, you should probably also add that suffix in the exemplary commands in the introductory comments at the top of that file.

Reply:
`${SUFFIX64}` is still wrong here.